yarn-project/aztec.js/src/contract_deployer/contract_deployer.ts

```diff
@@ -1,5 +1,5 @@
-import { Point } from '@aztec/circuits.js';
 import { ContractAbi } from '@aztec/foundation/abi';
+import { Point } from '@aztec/foundation/fields';
```

Comment:
there was some import error with `CompleteAddress` inside the box frontend apps, this was the fix (same change is in a few files)